content/en/observability_pipelines/troubleshooting.md

```diff
@@ -81,6 +81,35 @@ If you have configured your source to send logs to the Worker, make sure the por
 
 Run the command `netstat -anp | find "<port_number>"` to check that the port that the destination is listening on is not being used by another service.
 
+## Failed to connect error
+
+If you see an error similar to one of these errors:
+
+```
+Failed to connect to 34.44.228.240 port 80 after 56 ms: Couldn't connect to server
+```
+
+```
+connect to 35.82.252.23 port 80  failed: Operation timed out
+```
+
+```
+Failed to connect to ab52a1d16fxxxxxxxabd90c7526a1-1xxxx.us-west-2.elb.amazonaws.com port 80 after 225027 ms: Couldn't connect to server
+```
+
+And you:
+
+- Have a firewall between your source and your Workers, ensure traffic is allowed over your chosen port between the source and the Worker.
+- Have a firewall between the Workers and your destination, make sure it allows traffic from your Workers to the destination over the defined port.
+
+You can test your connectivity with curl. For example, if you have a Datadog Agent source, the curl command looks something like this:
```

Comment:
```suggestion
You can test your connectivity to observability with curl from your source location provided you have shell access to the source machine. For example, if you have a Datadog Agent source, the curl command looks something like this:
```

Reply:
@ckelner when you say "connectivity to observability..." is `observability` referring to the Worker?

Reply:
Ah sorry, yes that was a typo on my part, I meant to put "observability pipelines endpoint"

Reply:
updated! Sorry May!